libkbfs/keybase_service_base.go

```diff
@@ -365,6 +365,11 @@ func (k *KeybaseServiceBase) PaperKeyCached(ctx context.Context,
 	k.log.CDebugf(ctx, "Paper key for %s cached", arg.Uid)
 
 	if k.getCachedCurrentSession().UID == arg.Uid {
+		err := k.config.KBFSOps().KickoffAllOutstandingRekeys()
```

Comment:
Move this below the comment about ignoring errors, or add a new one?

Reply:
Added a comment instead of moving since it's not really the same scenario as the one below which requires waiting on the returned channel.